doc/radosgw/rgw-cache.rst

```diff
@@ -1,5 +1,5 @@
 ==========================
-RGW Data caching and CDN
+ RGW Data Caching and CDN
```

Comment:
I thought we weren't doing the indent thing, suggest we just consistently don't.

Reply:
Thank you. I'll remove two `=` from the lines above & below instead. I considered this "a special case" since those lines were the right length for "space before title text".